dataeng/jobs/createJobsTest.groovy

```diff
@@ -0,0 +1 @@
+import static analytics.AnalyticsEmailOptin.job as TestJob
```

Comment:
I'm not sure what's going on. I created this file using vim and the same red blocks appear.